arches/app/datatypes/base.py

```diff
@@ -532,3 +532,9 @@ def validate_node(self, node):
         a GraphValidationError
         """
         pass
+
+    def build_dropdown_option(self, default_tile_value):
```

Comment:
It seems like this should have a more generic name in case something similar is needed for other datatypes. If there are possibly different kinds of transformations, then maybe this method needs an optional argument.